abr_control/controllers/path_planners/second_order.py

```diff
@@ -124,12 +146,37 @@ def generate_path(self, state, target_pos, plot=False):
 
             plt.show()
 
+    def generate_path_function(self, state, target_pos, runtime):
```

Comment:
generate path as a function of time

Reply:
what's the use case here?

Reply:
When we want a path planner that converges wrt some time limit, instead of time steps. This is what we use in dewolf